packages/issue-receiver/action.yml

```diff
@@ -0,0 +1,17 @@
+name: Probby Issue Receiver
+description: Create an issue for changes to problem-specs
+author: Sascha Mann <git@mail.saschamann.eu>
+
+branding:
+  icon: bell
+  color: purple
+
+inputs:
+  token:
+    description: The GH token used to create the issue
+    default: ${{ github.token }}
+    required: false
```

Comment:
This will work with the default token but in case someone wants to create a different user to open the issues for whatever reason, they can do that.